features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1610,7 +1610,7 @@
     "client_assertion_type",
     "client_assertion"
   ],
-
+  "actions.types.authentication.enable": true,
```

Comment:
same here. this should be in master already

Reply:
This config is for new authentication action type